modules/es-node-disk-flood-watermark-reached.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/troubleshooting-logging-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="es-node-disk-flood-watermark-reached_{context}"]
+= Elasticsearch node disk flood watermark reached
+
+Elasticsearch enforces a read-only index block on every index that has both of these conditions:
+
+* One or more shards are allocated to the node.
+* One or more disks exceed the https://www.elastic.co/guide/en/elasticsearch/reference/6.8/disk-allocator.html[flood stage].
+
+Use the following procedure to troubleshoot this alert.
+
+.Procedure
+
+. Get the disk space of the Elasticsearch node:
++
+[source,terminal]
+----
+for pod in `oc -n openshift-logging get po -l component=elasticsearch -o jsonpath='{.items[*].metadata.name}'`; do echo $pod; oc -n openshift-logging exec -c elasticsearch $pod -- df -h /elasticsearch/persistent; done
+----
+// check this - is this a command or output?
+
+. In the command output, check the `nodes.node_name.fs` field to determine the free disk space on that node.
+// example output?
+
+. Increase the disk space on all nodes. If increasing the disk space is not possible, try adding a new data node to the cluster, or decrease the total cluster redundancy policy.
+
+--
+include::snippets/es-check-disk-space.adoc[]
+--
+
+. Continue freeing up and monitoring the disk space. After the used disk space drops below 90%, unblock writing to this node by running the following command:
++
+[source,terminal]
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_all/_settings?pretty -X PUT -d '{"index.blocks.read_only_allow_delete": null}'
```

Comment:
```suggestion
oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query="_all/_settings?pretty" -X PUT -d '{"index.blocks.read_only_allow_delete": null}'
```

Reply:
Per https://redhat-internal.slack.com/archives/C02321AKYTT/p1696866637850469?thread_ts=1696524477.252959&cid=C02321AKYTT we will leave this as is

Reply:
https://redhat-internal.slack.com/archives/C02321AKYTT/p1696861449261119?thread_ts=1696524477.252959&cid=C02321AKYTT